config/config.go

```diff
@@ -99,6 +101,14 @@ type Config struct {
 	InitialDelay metav1.Duration `json:"initialDelay,omitempty"`
 }
 
+func (c Config) GetContainerRuntimeExecutor(labels labels.Labels) (string, error) {
+	name, err := c.ContainerRuntimeExecutors.Select(labels)
+	if name != "" {
+		return name, err
```

Comment:
Looks like err is always nil in this case so return `name, nil` would be more readable?

Reply:
I know it is odd looking, but I think it is correct.

Reply:
this is because it can return `"", nil` - i.e. it did not find a CRE, but there was not an error